app/controllers/organisations_controller.rb

```diff
@@ -0,0 +1,60 @@
+class OrganisationsController < ApplicationController
```

Comment:
I created this (and associated views and model) via `rails g scaffold Organisation name --no-view-specs --no-helper --no-routing-specs`. And then did a little tweaking.